test/appium/tests/test_transaction.py

```diff
@@ -3,9 +3,9 @@
 from tests.base_test_case import SingleDeviceTestCase, MultipleDeviceTestCase
 from tests import transaction_users, api_requests, get_current_time, transaction_users_wallet
 from selenium.common.exceptions import TimeoutException
-
+from views.web_views.web_view_utils import is_web_view_opened
```

Comment:
there is no definition of the method, only the import

Reply:
what do you mean? I use this method in `test_open_transaction_on_etherscan`